onfleet/exceptions.py

```diff
@@ -1,3 +1,13 @@
+import re
+
+options_re = re.compile(r'"(.*?)"')
```

Comment:
As far as I can tell, this just captures anything in double quotes — am I wrong? That seems way too brittle. Isn't there something more specific we can search for?